ETC.tsv

```diff
@@ -3,9 +3,9 @@ ETC_20150317_000002	$The Kepa is not a strong monster. However, attacking it rec
 ETC_20150317_000003	$Large Kepa
 ETC_20150317_000004	$There are people who try to eat the Large Kepa sometimes. Don't let them.
 ETC_20150317_000005	$Red Kepa
-ETC_20150317_000006	$Contrary to common awareness, it is Red Kepa which is more similar to the original Kepa
-ETC_20150317_000007	$Poisoned Kepa.
-ETC_20150317_000008	$Monster
+ETC_20150317_000006	$Contrary to common belief, it's Red Kepa that are more similar to the original Kepa.
```

Comment:
Well, I mean, when you write "it's the Red Kepa that are" (= plural), vs the old version that should have been changed to "it's the Red Kepa that **is**" (= singular). I don't mind either way, but as I said, we probably should decide on a consistent way to talk about them.